### PR TITLE
Update writing dockerfiles page

### PR DESCRIPTION
**Pull request by oyelekci**

### CODE REVIEW

File: docs/how-to-docs/write-dockerfiles.md [hunk 1]
```diff
@@ -1,30 +1,16 @@
 ## Writing Dockerfiles
 
 #### Dockerfile best practice
-We recommend using dockers excellent guidance for this!  
-https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/
+We recommend familiarising yourself with Docker's excellent [guidance](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices) on this topic.   
 
-#### Docker images to build from
-This document lists all of our docker base images that you can build from:
+It is often easier to build from an existing base image. To find such base images that are maintained by Home Office colleagues, you can search the [UKHomeOffice organisation](https://github.com/UKHomeOffice) on Github for repos starting with ‘docker-’ - e.g.: [docker-java11-mvn](https://github.com/UKHomeOffice/docker-java8-mvn)
```
Comment (tasharnvb) [inline docs/how-to-docs/write-dockerfiles.md]:
```suggestion
It is often easier to build from an existing base image. To find such base images that are maintained by Home Office colleagues, you can search the [UKHomeOffice organisation](https://github.com/UKHomeOffice) on Github for repos starting with ‘docker-’ - e.g.: [docker-java8-mvn](https://github.com/UKHomeOffice/docker-java8-mvn)
```
or
```suggestion
It is often easier to build from an existing base image. To find such base images that are maintained by Home Office colleagues, you can search the [UKHomeOffice organisation](https://github.com/UKHomeOffice) on Github for repos starting with ‘docker-’ - e.g.: [docker-java11-mvn](https://github.com/UKHomeOffice/docker-java11-mvn)
```

File: docs/how-to-docs/write-dockerfiles.md [hunk 2]
```diff
@@ -1,30 +1,16 @@
...
-#### Guidance on building new base images
-All base images should be built with a set of onbuild commands in them to make sure anything built on top of them will automatically update the base OS, for example:
+Please make sure that any base image that you maintain adheres to the best practices set out by Docker, and includes instructions to update all existing packages - e.g.:
 ```
 yum install -y curl && yum clean all && rpm --rebuilddb
 ```
-The [nodejs base image](https://github.com/UKHomeOffice/docker-nodejs-base/blob/master/Dockerfile) is a good example of this.
+
+#### Home Office CentOS base image
+If none of the technology specific images work for you you can either build on top of them or build from the base [Centos image](https://github.com/UKHomeOffice/docker-centos-base).
```
Comment (tasharnvb) [inline docs/how-to-docs/write-dockerfiles.md]:
```suggestion
If none of the technology specific images work for you, you can either build on top of them or build from the base [Centos image](https://github.com/UKHomeOffice/docker-centos-base).
```

File: docs/how-to-docs/write-dockerfiles.md (hunk 1 above)
Comment (KashifSaadat) [inline docs/how-to-docs/write-dockerfiles.md]: java8 or java11?

Comment (oyelekci) [reply docs/how-to-docs/write-dockerfiles.md]: 11, that's one in demand these days

Comment (oyelekci) [reply docs/how-to-docs/write-dockerfiles.md]: ah sorry, fixing the link

**Pull request merged**